### PR TITLE
Efficiency upgrade, formatting change

### PR DESCRIPTION
**Pull request by Narrev**
Ready to merge

### CODE REVIEW

File: Engine/Nevermore.module.lua [hunk 1]
```diff
@@ -8,7 +8,7 @@ local ServerScriptService = game:GetService("ServerScriptService")
 
 -- Configuration
 local FolderName = "Modules" -- Name of Module Folder in ModuleRepositoryLocation
-local ModuleRepositoryLocation = ServerScriptService
```
Comment (devSparkle) [inline Engine/Nevermore.module.lua]: Why has the default `ModuleRepositoryLocation` been changed?

File: Engine/Nevermore.module.lua [hunk 2]
```diff
@@ -36,79 +36,52 @@ local function GetFirstChild(Parent, Name, Class) -- This is what allows the cli
...
-		end
-
-		Nevermore[FullName] = ResourceManager
-		return setmetatable(ResourceManager, SmartFolder)
-	else
-		error("[Nevermore] Attempt to index Nevermore with invalid key: string expected, got " .. type(FullName))
-	end
-end
-GetFolder = CreateResourceManager(Nevermore, "GetFolder")
-GetLocalFolder = CreateResourceManager(Nevermore, "GetLocalFolder")
```
Comment (devSparkle) [inline Engine/Nevermore.module.lua]: If `GetFolder` is intended for server-sided usage and `GetLocalFolder` is intended for local usage, shouldn't we just use `RunService` to determine if the function is being called from the server or the client?

File: Engine/Nevermore.module.lua (hunk 1 above)
Comment (Narrev) [inline Engine/Nevermore.module.lua]: I can change that back. That was my bad

File: Engine/Nevermore.module.lua (hunk 2 above)
Comment (Narrev) [inline Engine/Nevermore.module.lua]: GetLocalFolder is for instancing local storage on either the client or server. On the client, this is the LocalPlayer.Resources folder. On the server, this is the ServerStorage.Resources folder

**Comment by Narrev** [comment]
What else needs to be done to merge?

**Pull request merged**